content/docs/04-clusters/02-data-center/01-maas/05-install-manage-maas-pcg.md

```diff
@@ -43,18 +43,272 @@ As the following diagram shows, Palette provides an installer in the form of a D
 ![An architecture diagram of MaaS with PCG.](/clusters_maas_install-manage-mass-pcg_diagram-of-mass-with-pcg.png)
 
 
+# Install PCG
+
+Use the following steps to install a PCG cluster in your MAAS environment. You can use the [Palette CLI](/palette-cli) or the PCG Installer Image to deploy a PCG cluster. Review the prerequisites for each option to help you identify the correct install method.
+
 <br />
 
 
+<Tabs>
+
+<Tabs.TabPane tab="Palette CLI" key="palette-cli">
+
+
+## Prerequisites
+
+
+- Palette version 4.0.X or greater.
+
+
+- Canonical [MAAS installed](https://maas.io/docs/how-to-install-maas), set up, and available in your environment.
+
+
+- Download the Palette CLI from the [Downloads](/spectro-downloads#palettecli) page and install the CLI. Refer to the [Palette CLI Install](/palette-cli/install-palette-cli) guide to learn more.
+
+
+- A Palette API key. Refer to the [Create API Key](/user-management/user-authentication#apikey) page for guidance.
+
+  <br />
+
+  <WarningBox>
+
+  The installation does not work with Single Sign-On (SSO) credentials. You must use an API key from a local tenant admin account in Palette to deploy the PCG. After the PCG is configured and functioning, this local account is no longer used to keep the PCG connected to Palette, so you can disable the account if desired.
+
+  </WarningBox>
+
```

Comment:
@karl-cardenas-coding Aren't our warnings supposed to display as a yellow-ish box? This shows blue like an InfoBox even though it's correctly tagged. 🤔

Reply:
It's yellow in Netlify.